volto/src/addons/volto-govuk-theme/src/customizations/volto/components/theme/Search/Search.jsx

```diff
@@ -231,66 +251,25 @@ class Search extends Component {
                       defaultMessage="results"
                     />
                   </div>
-                  {/* <Header
+                  { <Header
                     style={{
                       marginTop: 0,
                     }}
                   >
                     <Header.Content className="header-content">
-                      <div className="govuk-body-xl sort-by">
-                        <FormattedMessage
-                          id="Sort By:"
-                          defaultMessage="Sort by:"
-                        />
+                      <div class="govuk-form-group">
```

Comment:
Because we are using React you should change 'class' into 'className', otherwise it'll throw a 'invalid dom' error.
Govuk designs assumes everyone is using basic html, so needs to be updated when brought across.